charts/spire/README.md

```diff
@@ -62,6 +62,32 @@ helm upgrade --install -n spire-mgmt spire-crds spire-crds --repo https://spiffe
 helm upgrade --install -n spire-mgmt spire spire --repo https://spiffe.github.io/helm-charts-hardened/ -f your-values.yaml
 ```
 
+### Running as Deployment (Stateless)
```

Comment:
This looks pretty good, but I think it may be a bit premature to promote an experimental feature in the instructions here. Can we split this out to a follow on PR for after its matured a bit?